doc/input-output-reference/src/overview/group-variable-refrigerant-flow-equipment.tex

```diff
@@ -1153,6 +1153,10 @@ \subsubsection{Outputs}
 
 Note: refer to the rdd file after a simulation for exact output variable names
 
+\paragraph{VRF Heat Pump Total Heating Rate {[}W{]}}\label{vrf-heat-pump-total-heating-rate-w}
```

Comment:
This needs a unique label (there are duplicate label warnings when file changes are viewed in github).

Reply:
Good catch. I'll add a suffix to it